tests/kernel/install_ltp.pm

```diff
@@ -286,11 +286,16 @@ sub get_default_pkg {
 sub install_from_repo {
     my @pkgs = split(/\s* \s*/, get_var('LTP_PKG', get_default_pkg));
 
-    zypper_call("in --recommends " . join(' ', @pkgs));
+    if (is_transactional) {
+        assert_script_run("transactional-update -n -c pkg install" . join(' ', @pkgs), 90);
+    } else {
+        zypper_call("in --recommends " . join(' ', @pkgs));
+    }
 
+    my $run_cmd = is_transactional ? 'transactional-update -c -d --quiet run' : '';
     for my $pkg (@pkgs) {
         my $want_32bit = $pkg =~ m/32bit/;
-        record_info("LTP pkg: $pkg", script_output("rpm -qi $pkg | tee "
+        record_info("LTP pkg: $pkg", script_output("$run_cmd rpm -qi $pkg | tee "
```

Comment:
This `rpm` command should be a read-only query. Does it need to be wrapped in `transactional-update`?